e2e/jest.test-setup.ts

```diff
@@ -7,7 +7,7 @@ beforeEach(async () => {
   await page.setUserAgent(userAgent);
   // Refer to https://github.com/puppeteer/puppeteer/blob/master/docs/api.md#pagesetdefaultnavigationtimeouttimeout
   await page.setDefaultNavigationTimeout(70000);
-  await page.setDefaultTimeout(10000);
+  await page.setDefaultTimeout(50000);
```

Comment:
I changed it to 10s but it has been the source for couple failed runs today. Making timeOut bigger.